.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - beta
+  - nightly
```

Comment:
Maybe we should just add beta too once and for all. Like this, all three stages will be tested. That's not very useful but whatever...